src/components/App/App.jsx

```diff
@@ -10,10 +10,8 @@ import Login from '../Login/Login.jsx';
 import ErrorPage from '../ErrorPage/ErrorPage.jsx';
```

Comment:
Since we have no props for App, we should remove the `import PropTypes from 'prop-types';` from line 3.